src/i18n/en-us/privacy.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  privacyIntro: 'Unsere Datenschutzerklärung',
+  privacyTitle: 'DATENSCHUTZERKLÄRUNG'
```

Comment:
German in English file?

Reply:
Wir müssten das ganze File übersetzen. Ist nur damit man nicht die Filler sieht :-D